_posts/2017-02-16-You-Want-The-Impossible.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title:  "You Want the Impossible"
+subtitle: "Web Development and Windows 10"
+date: 2017-02-16 13:12:32
+categories: development
+author: ryan
+show_avatar: true
+feature_image: /post-assets/impossible/ImpossibleLarge
+show_related_posts: false
+square_related: /post-assets/impossible/ImpossibleSquare
+blurb: "Web Development and Windows 10"
+comments: true
+headerDark: true
+permalink: /you-want-the-impossible/
+published: true
+---
+
+For as long as I have been doing web development, I have always had a Mac. It seemed like what everyone who did any Ruby programming used. I would only hear horror stories of anyone attempting to port their skills to the Windows operating system of the day. That is exactly where I find myself now. For the first time in over a decade, I own a Windows machine. I needed to find a way to work on the various Ruby, Node, and Elixir projects that encompass my day in a harsh and unknown terrain.
+
+## The Requirements
+
+Besides the aforementioned languages, I also needed a text editor. I have been a big fan of VIM for a couple of years and my workflow relies on it and the [setup that I have cobbled together.](https://github.com/ryanbillingsley/dotfiles) So I needed to be able to edit in VIM and then run the services in a way that is reliable and fast.
+
+There were also some nice-to-haves I wanted as well. I use a 256 color theme for VIM and I have grown quite attached to it, so I wanted [256 xterm color](https://commons.wikimedia.org/wiki/File:Xterm_256color_chart.svg) support. To go along with the theme, I also use [Inconsolas with Powerline](https://github.com/powerline/fonts/tree/master/Inconsolata) for my font. It is basically a patched version of Inconsolas that adds some additional symbols, that when combined with [AIrline](https://github.com/vim-airline/vim-airline), make the whole presentation look a little bit nicer.
+
+## The Candidates
+
+I knew there were a lot of avenues I could pursue with none of them being necessarily the “right” one. I limited my selection down to just a few so I could manage my findings. I started with [Bash on Ubuntu on Windows](https://msdn.microsoft.com/en-us/commandline/wsl/about).
```

Comment:
@ryanbillingsley you probably want to do a find and replace for fancy quotes with escaped characters. same for apostrophes